<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Focusing Camera View</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fiber end alignment monitoring</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="camera_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QSlider" name="speed_slider">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>63</number>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="tickPosition">
          <enum>QSlider::TicksAbove</enum>
         </property>
         <property name="tickInterval">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="arrows_widget" native="true">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <widget class="QPushButton" name="button_up">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>31</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_top.png</normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_top.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_down">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>60</y>
            <width>31</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_bottom.png</normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_bottom.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_right">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>30</y>
            <width>31</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_right.png</normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_right.png</iconset>
          </property>
         </widget>
         <widget class="QPushButton" name="button_left">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>30</y>
            <width>31</width>
            <height>28</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_left.png</normaloff>../../../../pynta/pynta/view/GUI/Icons/arrow_left.png</iconset>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="GraphicsLayoutWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>39</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
